package.json

```diff
@@ -12,6 +12,7 @@
     "e2e:ia": "npx codeceptjs run -c pr.codecept.js --grep '@ia'",
     "e2e:stt": "npx codeceptjs run -c pr.codecept.js --grep '@stt'",
     "e2e:dbaas": "npx codeceptjs run -c pr.codecept.js --grep '@dbaas'",
+    "e2e:admin": "npx codeceptjs run -c pr.codecept.js --grep '@admin'",
```

Comment:
please add the description of new tag to the read.me file.

Reply:
I think we don't need this script